dipy/reconst/dki.py

```diff
@@ -1266,23 +1503,48 @@ def rk(self, min_kurtosis=0, max_kurtosis=3):
             \frac{(\lambda_1+\lambda_2+\lambda_3)^2}{(\lambda_2-\lambda_3)^2}
             \left ( \frac{\lambda_2+\lambda_3}{\sqrt{\lambda_2\lambda_3}}-2
             \right )
+
+        References
+        ----------
+        .. [1] Tabesh, A., Jensen, J.H., Ardekani, B.A., Helpern, J.A., 2011.
+               Estimation of tensors and tensor-derived measures in diffusional
+               kurtosis imaging. Magn Reson Med. 65(3), 823-836
+        .. [2] Barmpoutis, A., & Zhuo, J., 2011. Diffusion kurtosis imaging:
+               Robust estimation from DW-MRI using homogeneous polynomials.
+               Proceedings of the 8th {IEEE} International Symposium on
+               Biomedical Imaging: From Nano to Macro, ISBI 2011, 262-265.
+               doi: 10.1109/ISBI.2011.5872402
         """
         return radial_kurtosis(self.model_params, min_kurtosis, max_kurtosis)
 
+    def kmax(self, sphere='symmetric362', gtol=1e-5, mask=None):
+        r""" Computes the maxima value of a single voxel kurtosis tensor
```

Comment:
This is not for a single voxel, is it? 

Also, there's got to be a way to propagate the mask from the fitting procedure, right? For example, you can look for voxels that have non-zero values of the parameters?